vault/datadog_checks/vault/vault.py

```diff
@@ -324,3 +331,10 @@ def _set_header(self, http_wrapper, header, value):
     def get_scraper_config(self, instance):
         # This validation is called during `__init__` but we don't need it
         pass
+
+    def transform(self, metric, scraper_config, transformerkey):
```

Comment:
can we use a more specific name for this? In case future transformers have different purposes. See example in kube_dns https://github.com/DataDog/integrations-core/blob/master/kube_dns/datadog_checks/kube_dns/kube_dns.py#L104-L122

Reply:
That makes sense. I changed the name to transform_route_metrics, as several metrics are using this same function. But let me know if you think I could have made a better name.